core/templates/domain/objects/NumberWithUnitsObjectFactorySpec.ts

```diff
@@ -32,7 +31,7 @@ describe('NumberWithUnitsObjectFactory', () => {
 
     beforeEach(() => {
       nwuof = new NumberWithUnitsObjectFactory(
-        new UnitsObjectFactory(), new FractionObjectFactory());
+        new UnitsObjectFactory());
```

Comment:
You can do `nwuof = new NumberWithUnitsObjectFactory(new UnitsObjectFactory());` here.

Reply:
Done